support/scalajson/src/main/scala/sjsonnew/support/scalajson/unsafe/CompactPrinter.scala

```diff
@@ -26,15 +26,15 @@ import shaded.scalajson.ast.unsafe._
  */
 trait CompactPrinter extends JsonPrinter {
 
-  def print(x: JValue, sb: StringBuilder): Unit = {
+  def print(x: JValue, sb: Appendable): Unit = {
```

Comment:
I though this was a bincompatible change since StringBuilder is an Appendable... What should I do? Add deprecated methods with StringBuilder? Or are these false positives?

Reply:
I'm guessing that mima here is correct because `CompactPrinter` is a trait, and someone could have extended it with `def print(x: JValue, sb: StringBuilder): Unit = ...`. We can I guess say it's a breaking change, assuming no one actually has extended `CompactPrinter` trait.

Reply:
and doing something like would not work either, right? because the extender is only overriding one of the methods...
```suggestion
  def print(x: JValue, sb: StringBuilder): Unit = print(x, sb: Appendable)
  def print(x: JValue, sb: Appendable): Unit = {
```

Reply:
I'm not sure. That might be enough to get ok from mima.

Reply:
silencing these in mima would be an option?

Reply:
Yea. We can put a warning in release note saying we broke a trait. That's ok with me.